include/dxc/DXIL/DxilShaderFlags.h

```diff
@@ -253,8 +258,6 @@ class ShaderFlags {
       m_bUAVLoadAdditionalFormats : 1; // SHADER_FEATURE_TYPED_UAV_LOAD_ADDITIONAL_FORMATS
   unsigned
       m_bLevel9ComparisonFiltering : 1; // SHADER_FEATURE_LEVEL_9_COMPARISON_FILTERING
-                                        // SHADER_FEATURE_11_1_SHADER_EXTENSIONS
-                                        // shared with EnableMSAD
```

Comment:
Not sure why this is here. I only see `m_bLevel9ComparisonFiltering` being used to set `SHADER_FEATURE_LEVEL_9_COMPARISON_FILTERING`
[`DxilShaderFlags.cpp:L88`](https://github.com/microsoft/DirectXShaderCompiler/blob/5ff9cbc7cb83ab2d5f52255412f61ac3226c4a08/lib/DXIL/DxilShaderFlags.cpp#L88)